samples/DataLoaderWithEFCore/DataLoaderWithEFCore/GraphApi/Schema/OutputTypes/Actor.cs

```diff
@@ -1,27 +1,24 @@
-using System;
-using System.Threading.Tasks;
 using AutoMapper;
 using DataLoaderWithEFCore.Data.Repositories;
 using GraphQL.Conventions;
 using GraphQL.DataLoader;
 using Models = DataLoaderWithEFCore.Data.Models;
 
-namespace DataLoaderWithEFCore.GraphApi.Schema
+namespace DataLoaderWithEFCore.GraphApi.Schema;
+
+public sealed class Actor
 {
-    public sealed class Actor
-    {
-        public Guid Id { get; set; }
+    public Guid Id { get; set; }
 
-        public string Name { get; set; }
+    public string Name { get; set; }
 
-        public string CountryCode { get; set; }
+    public string CountryCode { get; set; }
 
-        public Guid MovieId { get; set; }
+    public Guid MovieId { get; set; }
 
-        public async Task<Country> Country([Inject] IMapper mapper, [Inject] ICountryRepository repository, [Inject] DataLoaderContext dataLoaderContext)
-        {
-            var loader = dataLoaderContext.GetOrAddBatchLoader<string, Models.Country>("Actor_Country", repository.GetCountries);
-            return mapper.Map<Country>(await loader.LoadAsync(CountryCode).GetResultAsync());
-        }
+    public async Task<Country> Country([Inject] IMapper mapper, [Inject] ICountryRepository repository, [Inject] IDataLoaderContextAccessor dataLoaderContextAccessor)
+    {
+        var loader = dataLoaderContextAccessor.Context.GetOrAddBatchLoader<string, Models.Country>("Actor_Country", repository.GetCountries);
```

Comment:
Here is the only notable code change, to fit with the recommended practice within the GraphQL.NET documentation.

Reply:
Notice `[Inject] DataLoaderContext` --> `[Inject] IDataLoaderContextAccessor`